### DISCUSSION TITLE
docker/setup-qemu-action@v3 and macos-latest runners

### DISCUSSION THREAD

**Discussion by armingjazi** [docker/setup-qemu-action#137] [Q&A]
Hey people

I am using the ```docker/setup-qemu-action@v3``` with macos-latest runners

and getting following error:

Error: Unable to locate executable file: docker. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.

any ideas what can be done? building arm64 images on ubuntu-latest takes forever because of the emulation

**Comment by cclauss** [comment]
https://www.docker.com/blog/docker-desktop-for-mac-qemu-virtualization-option-to-be-deprecated-in-90-days
